coriolis_openstack_utils/resource_utils/users.py

```diff
@@ -17,6 +17,10 @@ def get_user(openstack_client, user_id):
     return openstack_client.keystone.users.get(user_id)
 
 
+def find_user(openstack_client, **kwargs):
```

Comment:
Will we ever need to add logic to this operation in the foreseeable future?
If not, I'd suggest we just call the client directly where we need this.
Good 'smell test': if you can just imagine writing `find_user  = openstack_client.keystone.users.find` and the code is equivalent, it means `find_user` should never have been defined...